resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesConf.scala

```diff
@@ -94,6 +96,20 @@ private[spark] class KubernetesDriverConf(
     custom.getOrElse(KubernetesConf.getResourceNamePrefix(appName))
   }
 
+  val driverServiceName: String = {
```

Comment:
Shall we use `lazy val` since this is not used `KubernetesDriverConf`?

Reply:
addressed in https://github.com/apache/spark/pull/46276